alot/buffers/thread.py

```diff
@@ -297,6 +294,35 @@ def focus_prev_unfolded(self):
         self.focus_property(lambda x: not x.is_collapsed(x.root),
                             self._tree.prev_position)
 
+    def focus_next_thread(self):
+        """focus next thread of the search result"""
+        self._update_thread('down')
+
+    def focus_prev_thread(self):
+        """focus previous thread of the search result"""
+        self._update_thread('up')
+
+    def _update_thread(self, movement):
+        if not self._search_buffer or not self._search_buffer():
+            return
+        search_buffer = self._search_buffer()
+        search_buffer.listbox.keypress((1, 1), movement)
```

Comment:
It'd be great not to modify internals of `search_buffer` here and instead add a public method there and call it here..
I believe the implementation would simply be to call the `set_focus` method in the `IterableWalker` `search_buffer.threadlist`.